src/applications/caregivers/tests/00.caregivers.cypress.spec.js

```diff
@@ -0,0 +1,17 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+import manifest from '../manifest.json';
+import environments from 'site/constants/environments';
+
+describe('Caregivers test', () => {
+  it('Loads the page body', () => {
+    if (
+      !(
+        manifest.e2eTestsDisabled &&
+        process.env.BUILDTYPE !== environments.LOCALHOST
+      )
+    )
```

Comment:
Can we add a comment that explains this block?

Reply:
Added!